### PR TITLE
fix: #1942 (presentproof) unable to correlate requests to responses

### LINKED ISSUE #1942: presentproof: unable to correlate requests to responses
**What I'm trying to do**

I am trying to correlate an inbound `presentation` message to a `request-presentation` sent previously.

**Expected result**

I expected to correlate the two messages using the [threading](https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0008-message-id-and-threading/README.md) standard.

**Actual result**

`presentproof.Client.SendRequestPresentation()` does not return the `threadID` and `presentproof.RequestPresentation` does not have an `ID` attribute.

### PR DESCRIPTION
**Pull request by llorllale**
closes #1942

Signed-off-by: George Aristy <george.aristy@securekey.com>

### CODE REVIEW

**Comment by llorllale** [comment]
low coverage is due to an unreachable line of code: https://github.com/hyperledger/aries-framework-go/pull/1944/files#diff-aee46ff9ba31682b63c038412a5d5f74R229

The msg's threadID is set prior to the above line here: https://github.com/hyperledger/aries-framework-go/blob/5b2b831914d8b1755438bd7822cf1e4827851ed2/pkg/didcomm/protocol/presentproof/service.go#L248-L247

File: pkg/controller/command/presentproof/command.go [hunk 1]
```diff
@@ -164,7 +164,7 @@ func (c *Command) SendRequestPresentation(rw io.Writer, req io.Reader) command.E
 		return command.NewValidationError(InvalidRequestErrorCode, errors.New(errEmptyRequestPresentation))
 	}
 
-	if err := c.client.SendRequestPresentation(args.RequestPresentation, args.MyDID, args.TheirDID); err != nil {
+	if _, err := c.client.SendRequestPresentation(args.RequestPresentation, args.MyDID, args.TheirDID); err != nil {
```
Comment (rolsonquadras) [inline pkg/controller/command/presentproof/command.go:167]: Shouldn't we support the correlation in controller as well ? Looks like this PR just addresses the SDK binding.

Comment (llorllale) [reply pkg/controller/command/presentproof/command.go:167]: The existing `Actions` stuff is more appropriate for the controller.

Comment (rolsonquadras) [reply pkg/controller/command/presentproof/command.go:167]: Actions is an array. Not sure how to correlate the request. In the [bdd tests](https://github.com/hyperledger/aries-framework-go/blob/master/test/bdd/pkg/presentproof/presentproof_controller_steps.go#L262-L276) we are taking the first element.

Comment (llorllale) [reply pkg/controller/command/presentproof/command.go:167]: @rolsonquadras added TODO

**Pull request merged**